### PR TITLE
Added mongodb implementation

### PR DESCRIPTION
**Pull request by dominikdysput**

### CODE REVIEW

File: Storage/Stateflows.Storage.MongoDb/Utils/CollectionNames.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Stateflows.Storage.MongoDB.Utils
+{
+    internal static class CollectionNames
+    {
+        public const string Context = "Context";
```
Comment (mikolaj-milewski) [inline Storage/Stateflows.Storage.MongoDb/Utils/CollectionNames.cs:5]: Use '_v1' suffix for those

File: Storage/Stateflows.Storage.MongoDb/Stateflows/MongoDBStorage.cs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+                    StateflowsJsonConverter.SerializeObject(timeToken)
+                );
+                tokens.Add(timeToken, timeTokenEntity);
+            }
+
+            await collection.InsertManyAsync(tokens.Values);
+            UpdateIdsForInsertedTimeTokens(tokens, timeTokens);
+        }
+
+        public async Task ClearTimeTokens(BehaviorId behaviorId, IEnumerable<string> ids)
```
Comment (mikolaj-milewski) [inline Storage/Stateflows.Storage.MongoDb/Stateflows/MongoDBStorage.cs:62]: I don't know why, but it doesn't clear old time tokens - yes, I've tested it ;-)

Comment (dominikdysput) [reply Storage/Stateflows.Storage.MongoDb/Stateflows/MongoDBStorage.cs:61]: Could you check it now? :)

File: Storage/Stateflows.Storage.MongoDb/Utils/CollectionNames.cs (hunk 1 above)
Comment (dominikdysput) [reply Storage/Stateflows.Storage.MongoDb/Utils/CollectionNames.cs]: Changed.

File: Storage/Stateflows.Storage.MongoDb/Stateflows/MongoDBStorage.cs (hunk 2 above)
Comment (mikolaj-milewski) [reply Storage/Stateflows.Storage.MongoDb/Stateflows/MongoDBStorage.cs:61]: Works ;-)

**Pull request merged**